README.md

```diff
@@ -57,6 +57,12 @@ target_link_libraries(<your project name> PRIVATE Azure::azure-storage-blobs)
 
 You can reference this [vcpkg Quick Start](https://github.com/microsoft/vcpkg#quick-start-windows) for more details.
 
+#### Getting Beta Releases in Vcpkg
+
+Official vcpkg registry may have beta versions of Azure SDK client libraries, up until a given library gets released as General Availability (GA) version. After that, we don't publish post-GA beta releases of that library in the official registry.

```

Comment:
We should use "stable"/"beta" instead of GA/preview.

Please fix the wording to use stable here.